tests/conftest.py

```diff
@@ -6,13 +6,36 @@
 import sys
 import time
 
+import json
 import pytest
 
 from jupyterhub_traefik_proxy import TraefikEtcdProxy
 from jupyterhub_traefik_proxy import TraefikConsulProxy
 from jupyterhub_traefik_proxy import TraefikTomlProxy
 
 
+@pytest.fixture
+async def autohttps_toml_proxy():
+    """Fixture returning a configured Let's Encrypt TraefikTomlProxy"""
+    proxy = TraefikTomlProxy(
+        public_url="https://jupyter.test:8443",
+        traefik_api_password="admin",
+        traefik_api_username="api_admin",
+        should_start=True,
+        traefik_auto_https=True,
+        traefik_letsencrypt_email="jovyan@jupyter.test",
+        traefik_letsencrypt_domains=["jupyter.test"],
+        traefik_acme_server="https://0.0.0.0:14000/dir",
```

Comment:
Is this correct? Is this the configuration of where traefik is supposed to find the ACME server? Then we need an actual address I think. I think if we have started Pebble locally, then we should write `localhost` here. We could write 127.0.0.1 but then our host header with the request would be 127.0.0.1 instead of localhost, and that is bad for the TLS certificate pebble will have which allows for `localhost` but not `127.0.0.1` I think.

Reply:
From the logs, it looks like the interaction between traefik and the acme server happens ok and they find each other, but I'll switch to localhost to give it an actual address and be specific.